src/super_gradients/training/transforms/processing.py

```diff
@@ -240,5 +282,49 @@ def default_ppyoloe_coco_processing_params() -> Tuple[List[str], Processing]:
             ImagePermute(permutation=(2, 0, 1)),
         ]
     )
-    class_names = COCO_DETECTION_CLASSES_LIST
-    return class_names, image_processor
+
+    params = dict(
+        class_names=COCO_DETECTION_CLASSES_LIST,
+        image_processor=image_processor,
+        iou=0.65,
+        conf=0.5,
+    )
+    return params
+
+
+def default_deciyolo_coco_processing_params() -> dict:
+    """Processing parameters commonly used for training DeciYolo on COCO dataset.
+    TODO: remove once we load it from the checkpoint
+    """
+
+    image_processor = ComposeProcessing(
+        [
+            ReverseImageChannels(),
```

Comment:
Like we said- its the other way around (since we load with PIL its already RGB).
Remove ReverseImageChannels() from here and add it to the other detection preprocessing pipeplines please.

Reply:
Changed in 21b5534c31cf9079e67b6201a2081764ac156d8a